strategies/iCARL/experiment.py

```diff
@@ -0,0 +1,191 @@
+        config.lr_milestones = [49, 63]
+        config.lr_factor = 5.
+        config.wght_decay = 0.00001
+        # class incremental learning: classes mutual exclusive
+        config.fixed_class_order = [87, 0, 52, 58, 44, 91, 68, 97, 51, 15,
+                                    94, 92, 10, 72, 49, 78, 61, 14, 8, 86,
+                                    84, 96, 18, 24, 32, 45, 88, 11, 4, 67,
+                                    69, 66, 77, 47, 79, 93, 29, 50, 57, 83,
+                                    17, 81, 41, 12, 37, 59, 25, 20, 80, 73,
+                                    1, 28, 6, 46, 62, 82, 53, 9, 31, 75,
+                                    38, 63, 33, 74, 27, 22, 36, 3, 16, 21,
+                                    60, 19, 70, 90, 89, 43, 5, 42, 65, 76,
+                                    40, 30, 23, 85, 2, 95, 56, 48, 71, 64,
+                                    98, 13, 99, 7, 34, 55, 54, 26, 35, 39]
+        config.seed = 2222
+
+        #
+        device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
+        torch.manual_seed(config.seed)
+        torch.cuda.manual_seed(config.seed)
+        np.random.seed(config.seed)
+        random.seed(config.seed)
+        torch.backends.cudnn.enabled = False
+        torch.backends.cudnn.deterministic = True
+
+        per_pixel_mean = get_dataset_per_pixel_mean(
+            CIFAR100(expanduser("~") + "/.avalanche/data/cifar100/",
+                     train=True, download=True,
+                     transform=transforms.Compose([transforms.ToTensor()])))
+
+        transforms_group = dict(
+            eval=(transforms.Compose([
+                transforms.ToTensor(),
+                lambda img_pattern: img_pattern - per_pixel_mean]), None),
+            train=(transforms.Compose([
+                transforms.ToTensor(),
+                lambda img_pattern: img_pattern - per_pixel_mean,
+                icarl_cifar100_augment_data]), None),
+        )
+
+        train_set = CIFAR100(expanduser("~") + "/.avalanche/data/cifar100/",
+                             train=True, download=True, )
+        test_set = CIFAR100(expanduser("~") + "/.avalanche/data/cifar100/",
+                            train=False, download=True, )
+
+        train_set = AvalancheDataset(train_set,
+                                     transform_groups=transforms_group,
+                                     initial_transform_group='train')
+        test_set = AvalancheDataset(test_set,
+                                    transform_groups=transforms_group,
+                                    initial_transform_group='eval')
+
+        scenario = nc_benchmark(
+            train_dataset=train_set,
+            test_dataset=test_set,
+            n_experiences=config.nb_exp,
+            task_labels=False, seed=config.seed,
+            shuffle=False,
+            fixed_class_order=config.fixed_class_order)
+
+        evaluator = EvaluationPlugin(ExperienceAccuracy(),
```

Comment:
this is ok, but can you use `accuracy_metrics(experience=True, stream=True)`? It is the recommended way to build metrics in avalanche.

Reply:
DONE